proto/resource_usage_agent.proto

```diff
@@ -14,9 +14,15 @@ option java_package = "org.tikv.kvproto";
 
 // ResourceUsageAgent is the service for storing resource usage records.
 service ResourceUsageAgent {
+    // DEPRECATED: We now use `Report` to report not only CPU time.
+    //
     // Report the CPU time records. By default, the records with the same
     // resource group tag will be batched by minute.
     rpc ReportCPUTime(stream CPUTimeRecord) returns (EmptyResponse) {}
```

Comment:
We can change CPUTimeRecord to ResourceUsageRecord, since this change is compatible.

Reply:
I’m not sure before, it seems that the type name of the parameter does not affect compatibility?

Reply:
My idea is to keep the original proto text unchanged to avoid the illusion of breaking compatibility, and these will be deleted when the transition is completed.